mlrun/utils/helpers.py

```diff
@@ -1628,3 +1655,12 @@ def format_alert_summary(
     result = result.replace("{{name}}", alert.name)
     result = result.replace("{{entity}}", event_data.entity.ids[0])
     return result
+
+
+def _reload(module):
```

Comment:
I suggest adding a `max_recursion_depth` parameter here.
I don't think the user should supply it though, so let's put a high enough (but not infinite) threshold in the `mlconf` and use that as the value. Maybe `100`?